src/network/networkd-link.c

```diff
@@ -1783,6 +1783,8 @@ int link_up(Link *link) {
 
         log_link_debug(link, "Bringing link up");
 
+        (void) link_ipv6_enabled(link);
```

Comment:
Maybe, is this a typo of `link_enable_ipv6()`?

Reply:
we could do that .